### ISSUE TITLE
Support for Legacy Factions

### ISSUE THREAD

**Issue by timbru31** [timbru31/SilkSpawners#91]
Yet another Factions 1.6 clone, but with changed imports. https://github.com/redstone/LegacyFactions/

@MarkehMe just my two cents:
1. Why don't you correctly fork the repos? Makes it hard to track Factions derivates.
2. Why change the package name? If plan to offer the same stable Factions 1.6 API, then current Factions support won't be broken because of new imports.

**Comment by MarkehMe** [comment]
Sure! No worries 😄 

Quickly - I've been [working on docs for the API](https://github.com/redstone/LegacyFactions/wiki/API), it's a lot of work though. I'm getting there.. slowly. 

You might find [FPlayerColl](https://github.com/redstone/LegacyFactions/blob/master/src/main/java/net/redstoneore/legacyfactions/entity/FPlayerColl.java) and [FactionColl](https://github.com/redstone/LegacyFactions/blob/master/src/main/java/net/redstoneore/legacyfactions/entity/FactionColl.java) helpful. 

In regards to no 1:
FactionsUUID has [so many issues](https://github.com/drtshock/Factions/issues) and [really isn't being worked on](https://github.com/drtshock/Factions/commits/1.6.x) by the maintainer anymore. I've actually made several contact attempts, even on Skype and Twitter to get things fixed and offer help. 😞 

I took it on myself to work on the project, and it's gotten to the point where it's not really a fork and GitHub found it eligible to remove it from a fork to it's main repo. The changes are pretty substantial, especially after refactoring - it's not worthwhile. 

In regards to no 1:
I suspect the developer is just dealing with life & work, thats ok, but it's pretty broken and I don't think people even realise. I found a lot of issues while building [FactionsFramework](https://github.com/MarkehMe/FactionsFramework).

LegacyFactions is a rewrite of the underlying 1.6 API while keeping the same _face_. There are a lot of issues that I've had to fix - there was a good reason Factions 2 was a major rewrite of Factions 1.

Most (if not all) of the API events are different, there is still a lot of under the good work to do though in terms of making it super developer friendly. [Land transactions](https://github.com/redstone/LegacyFactions/blob/master/src/main/java/net/redstoneore/legacyfactions/event/EventFactionsLandChange.java#L47-L53) was a major featured I needed personally. But I know there needs to be things like JavaDocs and whatnot put out - but it's all coming together in time :) 

**tl;dr Wow thats a lot of text. Sorry for the essay!** 

I hope that makes sense, and as you can tell I'm having a wild friday night in - so feel free to [join us on Discord](https://discord.gg/p3VjnUd) and ask questions or yell at me about how terrible this is 😆

**Comment by timbru31** [comment]
Thanks for the detailed explanation! 👍

**Comment by timbru31** [comment]
Closing since Legacy Factions seems unmaintained for the time being.
